reana_job_controller/config.py

```diff
@@ -26,6 +28,13 @@
 }
 """Supported job compute backends and corresponding management class."""
 
+JOB_MONITORS = {
+    'kubernetes': KubernetesJobsMonitor,
+    'htcondorcern': HTCondorJobsMonitorCERN
```

Comment:
```suggestion
    'htcondorcern': HTCondorCERNJobsMonitor
```
As everywhere else?

Reply:
Yup we talked to make the class hierarchy extensible for CERN or ND etc specifics via suffixes, see CondorCERN, CondorND musings in https://github.com/reanahub/reana-job-controller/pull/144#issuecomment-503081991

Personally I would advocate going full suffix way, like:

```
JobMonitor - abstract class
JobMonitorHTCondor - for HTCondor generics
JobMonitorHTCondorCERN - for additional CERN specificities
JobMonitorHTCondorND - for additional ND specificities
JobMonitorSlurm - for Slurm generics
JobMonitorSlurmCERN - for additional CERN Slurm specifics
...
```

Ditto for JobManager.

As a nice side effect this would help with finding backend specifics everywhere, e.g. `grep -ri htcondorcern`.